.htaccess

```diff
@@ -0,0 +1,32 @@
+<IfModule mod_rewrite.c>
+        SetEnv HTTP_MOD_REWRITE On
+        RewriteEngine On
+
+        # "Symlink" the root to the latest release (with "moved temporarily")
+        RewriteRule ^/?$ /srcdocs/3.6/ [R=302,L]
+
+        # Remove trailing slashes, ensures the redirect to srcdocs works
+        RewriteCond %{REQUEST_FILENAME} !-d
+        RewriteRule ^(.*)/$ /$1 [L,R]
+
+        # Lookup script (used to be a SS app, now just a simple script)
+        RewriteCond %{REQUEST_URI} ^/search/lookup$
+        RewriteRule ^search/lookup /search/lookup.php [L,R=301,QSA]
+
+        # "Symlink" a "current" folder to the latest release
+        RewriteRule ^.*/current.*?(.*)? /srcdocs/3.6/$1 [R=301,L]
+
+        # Rewrite trunk to master
+        RewriteRule ^.*/trunk/.*(.*)? /srcdocs/master/$1 [L,R=301]
```

Comment:
Not necessarily, but I'd like to avoid having some .htacces magic to rewrite the URL without it. Simply because I don't want the code base to end up in the state where the whole thing is held together with .htaccess, I felt a bit like this is how it was when I started looking at it on Friday, especially with htdocs being the old web root.